microsoft-automerge.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+      "github>gsoft-inc/renovate-config"
```

Comment:
To discuss (with Gérald) what we want the experience to be for the end user:

```
 "extends": [
      "github>gsoft-inc/renovate-config"
	    "github>gsoft-inc/renovate-config/microsoft-automerge.json" (more granular only enable for microsoft)
  ],
```


```
 "extends": [
	    "github>gsoft-inc/renovate-config/microsoft-automerge.json" (include default)
  ],
```

Reply:
We'll remove the extend to default config